scripts/run_backend_tests.py

```diff
@@ -309,27 +313,18 @@ def main(args=None):
     common.fix_third_party_imports()
 
     if parsed_args.generate_coverage_report:
-        python_utils.PRINT(
```

Comment:
Should we replace this with an equivalent check?

Reply:
Not sure why was this removed, added it back.